ibis/expr/api.py

```diff
@@ -1132,6 +1132,7 @@ def lead(arg, offset=None, default=None):
 rank = _unary_op('rank', ops.MinRank)
 dense_rank = _unary_op('dense_rank', ops.DenseRank)
 percent_rank = _unary_op('percent_rank', ops.PercentRank)
+cume_dist = _unary_op('cume_dist', ops.CumeDist)
```

Comment:
OT: I don't think we have any doc-strings for these, can you add an issue

This brings up the naming here is very non-obvious w/o a doc-string. Can you clarify waht this means and why we should use it.

Reply:
From https://docs.microsoft.com/en-us/sql/t-sql/functions/cume-dist-transact-sql?view=sql-server-ver15, CUME_DIST calculates the cumulative distribution of a value within a group of values. In other words, CUME_DIST calculates the relative position of a specified value in a group of values. Assuming ascending ordering, the CUME_DIST of a value in row r is defined as the number of rows with values less than or equal to that value in row r, divided by the number of rows evaluated in the partition or query result set. CUME_DIST is similar to the PERCENT_RANK function.

The current version (on master) of PERCENT_RANK was performing as a CUME_DIST, this PR moves PERCENT_RANK code to CUME_DIST and implement properly the PERCENT_RANK

Reply:
I will add a docstring there.

Reply:
I added a docstring to `CUME_DIST` (https://github.com/ibis-project/ibis/pull/2224/files#diff-4653676a8aecaa42fa30e24aa86db8a963e8fa912fb563c0b6ccbec340b5bb9aR1213) and created a issue about the missing docstrings (#2471)